var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -145,8 +145,8 @@ class Acts(CMakePackage, CudaPackage):
     depends_on('boost @1.62:1.69 +program_options +test', when='@:0.10.3')
     depends_on('boost @1.71: +filesystem +program_options +test', when='@0.10.4:')
     depends_on('cmake @3.14:', type='build')
-    depends_on('dd4hep @1.11:', when='+dd4hep')
-    depends_on('dd4hep @1.11: +geant4', when='+dd4hep +geant4')
+    depends_on('dd4hep @1.11: +dddetectors', when='+dd4hep')
+    depends_on('dd4hep @1.11: +dddetectors +ddg4', when='+dd4hep +geant4 +examples')
```

Comment:
```suggestion
    depends_on('dd4hep @1.11: +dddetectors +ddrec', when='+dd4hep')
    depends_on('dd4hep @1.11: +dddetectors +ddrec +ddg4', when='+dd4hep +geant4 +examples')
```
That's another option, per https://github.com/spack/spack/pull/27717#issuecomment-988635051.

Reply:
I'll adopt this version since it does not seem like dd4hep will change the reason for the conflict in the near future. This indeed makes it clearer.